odysseus/halow_setup/README.md

```diff
@@ -56,38 +57,181 @@ The following command line items are run inside the pi terminal.  If you reboot
 `sudo reboot now`
 
 ### Configure connection
-`cd ~/nrc_pkg/script/conf/US/`
+`cd ~/nrc_pkg/script/conf/`
 
-open `sta_halow_open.conf` and edit in the `network={}`
+#### As station (STA mode)
+open `US/sta_halow_open.conf` and edit in the `network={}`
 
-- `ssid=SilexAH` (or SSID of another AP)
+- `ssid=NER_Halow` (or SSID of another AP)
 - `scan_ssid=1`
 - `scan_freq= <channel>`
 - `freq_list= <channel>`
 
-See channels [here](https://github.com/newracom/nrc7292_sw_pkg/blob/master/package/doc/UG-7292-003-S1G_Channel.pdf), use the 2.4/5 ghz codes as linux doesn't understand sub-ghz.  Can use multiple channels by space-seperating the channel numbers.  **Ensure the AP is set to the correct channel!**  Optionally do not set this, and it will scan all channels to find the AP at ssid.
+See channels [here](https://github.com/newracom/nrc7292_sw_pkg/blob/master/package/doc/UG-7292-003-S1G_Channel.pdf), use the 2.4/5 ghz codes as linux doesn't understand sub-ghz.  Can use multiple channels by space-seperating the channel numbers.  **Ensure the AP is set to the correct channel!**  Optionally do not set this, and it will scan all channels to find the AP at ssid.  
+Example `sta_halow_open.conf`:  
+```
+country=US
+network={
+   ssid="NER_Halow"
+   scan_ssid=1
+   key_mgmt=NONE
+   scan_freq= 2412
+   freq_list= 2412
+}
+p2p_disabled=1
+ignore_old_scan_res=1
+```
+Now set a static IP to ensure stability and easy connection to AP-side programs (ex. Argos).  
```

Comment:
This all feels like A LOT for a readme, this stuff should probably go in our confluence, especially since it is private and you can format everything easier. Confluence uses markdown iirc so you can just copy paste a lot of this stuff over